crates/codegen/runtime/cargo/crate/src/runtime/bindings/mod.rs

```diff
@@ -1,22 +1,63 @@
+//! This module contains APIs used to resolve symbol bindings in source code. These APIs can be used
+//! for code analysis features such as "Go to Definition" or "Find References".
 #[path = "generated/binding_rules.rs"]
 mod binding_rules;
 
 use std::rc::Rc;
 
 use semver::Version;
+use visibility as _;
 
 use crate::cst::KindTypes;
 
+/// An utility for building a [`BindingGraph`] for a collection of source files.
 pub type BindingGraphBuilder = metaslang_bindings::BindingGraphBuilder<KindTypes>;
```

Comment:
nit typo: "an utility"..

Reply:
Also, isn't `BindingGraphBuilder` internal?

Reply:
> nit typo: "an utility"..

That's what it says... Are you looking into the latest changes?

Reply:
> That's what it says... Are you looking into the latest changes?

Ah, I thought you were saying that it should be "an", but it must be "a" instead.

Reply:
> That's what it says... Are you looking into the latest changes?

I think so? It should've been "a utility" not "an utility", right?

My understanding is that it is determined based on the sound, not the written letter. If it sounds like "Y" (consonant), it uses "a", like "a user". If it sounds like "U" (vowel), it uses "an", like "an umbrella"...

English is weird 🥲